FLIR/conservator/run_tasks_plugins/conservator_cli.py

```diff
@@ -0,0 +1,12 @@
+from FLIR.conservator.cli import main
+
+
+def conservator_cli(cmd, args=(), log="INFO", task=None):
+    log_arg = "--log=" + log
```

Comment:
I've always done `conservator --log DEBUG` without the `=`, didn't know this also worked!